### PR TITLE
feat: add tim's scripts

### PR DESCRIPTION
**Pull request by mitosagi**
## Description

Add the list of scripts by tim provided by @shumm7 . The reason for splitting the file is to reduce the size of the apm-data update.

Many thanks to Shu for providing so much package data.

## Related Issue

- closes https://github.com/team-apm/apm-data/issues/257

## How Has This Been Tested?

- Visually check the package list on apm.

## Checklist:

- [ ] ~Updated the modification date in `mod.xml`.~ To prevent conflicts from occurring, it will not update at this time.

### CODE REVIEW

File: v3/packages/tim.json [hunk 1]
```diff
@@ -0,0 +1,2791 @@
...
+      "id": "tim/simwrp2",
+      "name": "簡易ワープ(改)",
+      "overview": "画像を歪ませる",
+      "description": "簡易ワープの改良版。画像を歪ませます。",
+      "developer": "ティム",
+      "pageURL": "https://www.nicovideo.jp/watch/sm19322218",
+      "downloadURLs": ["https://tim3.web.fc2.com/sidx.htm#SimWrp2"],
+      "latestVersion": "2012/11/10",
+      "nicommons": "sm19322218",
+      "files": [{ "filename": "tim/簡易ワープV2.anm" }],
```
Comment (hal-shu-sato) [inline v3/packages/tim.json:2494]:
```suggestion
      "files": [{ "filename": "script/tim/簡易ワープV2.anm" }],
```

File: v3/packages/tim.json [hunk 2]
```diff
@@ -0,0 +1,2791 @@
...
+      "releases": [
+        {
+          "version": "2012/11/10",
+          "integrity": {
+            "archive": "sha384-bUHNWtc9SH0L7bcqbRwfMgZBGT+STuXPmjLocx8RtLe+DuMnVglFkWwVAMIQX04t",
+            "file": [
+              {
+                "target": "tim/簡易ワープV2.anm",
```
Comment (hal-shu-sato) [inline v3/packages/tim.json:2502]:
```suggestion
                "target": "script/tim/簡易ワープV2.anm",
```

File: v3/packages/tim.json [hunk 3]
```diff
@@ -0,0 +1,2791 @@
...
+        { "filename": "satsuki/@onsei.anm", "archivePath": "時系列音声波形" }
+      ],
+      "releases": [
+        {
+          "version": "2012/08/07",
+          "integrity": {
+            "archive": "sha384-xbZmgeAArDSSIEDj6rySMORq9MYDWFx5ZmQ/Iu9FnsoaNUO+ZtH1NY0if5fdK5Q6",
+            "file": [
+              {
+                "target": "satsuki/@onsei.anm",
```
Comment (hal-shu-sato) [inline v3/packages/tim.json:83]:
```suggestion
                "target": "script/satsuki/@onsei.anm",
```

File: v3/packages/tim.json [hunk 4]
```diff
@@ -0,0 +1,2791 @@
...
+            "archive": "sha384-9h1P7PkEJ1DvigkF8jTW6uuqfpYrDpaE5DYViF99RahqlUDhClJt10P+8ymPwjV4",
+            "file": [
+              {
+                "target": "script/tim/split1.txt",
+                "hash": "sha384-AfqU6YriWKpxd+rMm2iPNmD1BL6i3uOTKVE0rB6EPxvZyKCqQsoXEOesFzTtDHwq"
+              },
+              {
+                "target": "script/tim/split2.txt",
+                "hash": "sha384-F6m9zfBsFY0bENbpo7mdoZkHlNxdMRIG/BvGGl8YuqFITqCE+cp/bB2uJqHUYqgk"
+              },
```
[the remark this answers is not in the document]
Comment (mitosagi) [reply v3/packages/tim.json:1685]: Are these lines OK?

**Comment by mitosagi** [comment]
Thank you for the detailed view. Fixed.

File: v3/packages/tim.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (hal-shu-sato) [reply v3/packages/tim.json:1688]: Could you delete this data?
They are data files that can be used with the plugin.
I think they are not suited for detecting versions.

[the remark this answers is not in the document]
Comment (mitosagi) [reply v3/packages/tim.json]: You are certainly right.

**Review by hal-shu-sato — APPROVED** [review]
Thank you for adding them!
I also thank @shumm7 !!

**Pull request merged**